### DISCUSSION TITLE
Constraints on ND

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5082] [General]
Hi developers,

I figured out, that not all constraints are displayed on the ND. Is there a limitation from Airbus? I was diving into RWY 23R in UAAA (TUKTO1G).

![grafik](https://user-images.githubusercontent.com/76531893/121530741-02c0d480-c9fe-11eb-82eb-b100e1fb17c6.png)

![grafik](https://user-images.githubusercontent.com/76531893/121530777-0bb1a600-c9fe-11eb-90bd-a817485c61d4.png)

greets Klaus

**Comment by MapleNAV-TCASFAULT** [comment]
I believe that that is a flight plan manager issue unless airbus is like that but not from what I remember.... As always it will be fixed if it is an issue.
